### PR TITLE
#28 [FrontApp] Judge can assessment submited result per task

### PR DESCRIPTION
**Pull request by PyrinAndrii**
**Changes:**
- update seeds
- add / update specs
- update locales
- add assessments routes (new, create, edit, update)
- add tasks show for judges
- update task_assessment validation
- add task_assessments controller
- add task_assessments new and edit views

---
**Screenshots:**
Screenshot 2022-05-11 at 13 15 25

Screenshot 2022-05-11 at 13 15 51

### CODE REVIEW

File: spec/features/ui/assessments/create_spec.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  context 'without session' do
+    it 'does not show task submissions' do
+      visit "assessments/new?task_submission=#{submission.id}"
+
+      expect(page).not_to have_button 'Create'
+      expect(page).to have_content 'You need to sign in or sign up before continuing.'
+    end
+  end
+
+  context 'with session' do
```
Comment (sveredyuk) [inline spec/features/ui/assessments/create_spec.rb]: + test for non-judge members

File: spec/features/ui/assessments/update_spec.rb [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  context 'without session' do
+    it 'does not show task submissions' do
+      visit "assessments/#{assessment.id}/edit?task_submission=#{submission.id}"
+
+      expect(page).not_to have_button 'Update'
+      expect(page).to have_content 'You need to sign in or sign up before continuing.'
+    end
+  end
+
+  context 'with session' do
```
Comment (sveredyuk) [inline spec/features/ui/assessments/update_spec.rb]: + test for non-judge member

**Review by sveredyuk — CHANGES_REQUESTED** [review]
Well done! +few specs and we can merge it

File: spec/features/ui/assessments/update_spec.rb (hunk 2 above)
Comment (PyrinAndrii) [reply spec/features/ui/assessments/update_spec.rb]: done both in [add not judge`s flows specs for assessments](https://github.com/DevChallengeIT/devchallenge-platform/pull/50/commits/bb22cf9d916a48f5a13d415f14ee3149495ec2a0)

**Review by sveredyuk — APPROVED** [review]

**Pull request merged**